packages/core/src/kernelCoreTemplate.js

```diff
@@ -124,24 +124,32 @@
         if (shouldAvoidProp(propsToAvoid, prop)) {
           continue
         }
-        if (Object.getOwnPropertyDescriptor(globalRef, prop)?.configurable === false) {
+        let desc = Object.getOwnPropertyDescriptor(globalRef, prop)
+        if (desc?.configurable === true) {
+          desc = {
+            configurable: false,
+            set: () => {
+              console.warn(
+                `LavaMoat - property "${prop}" of globalThis cannot be set under scuttling mode. ` +
+                'To learn more visit https://github.com/LavaMoat/LavaMoat/pull/360.',
+              )
+            },
+            get: () => {
+              throw new Error(
+                `LavaMoat - property "${prop}" of globalThis is inaccessible under scuttling mode. ` +
+                'To learn more visit https://github.com/LavaMoat/LavaMoat/pull/360.',
+              )
+            },
+          }
+        } else if (desc?.writable === true) {
+          desc = {
+            configurable: false,
+            writable: false,
+            value: undefined,
```

Comment:
We could (but I'm not sure if it's the right thing to do) put an object here with a toString function throwing the error or returning `'[lavamoat inaccessible]'` so it'd be easier to figure out than if you just get undefined

Reply:
7c274a3